components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,126 @@
+
+It achieves optimal operational performance through either a single-node deployment or through a deployment across a cluster. In a single-node deployment, a single load-balancing controller performs all administrative functions and all analytics data gathering and processing. Whereas, in a high availability load balancing cluster, additional nodes provide node-level redundancy for the load-balancing controller and maximize performance for CPU-intensive analytics functions.
+
+### What happens during a failover?
+
+Chef Automate HA enables the ability of the system to continue functioning even after the failure of one or more servers. A part of HA is failover, which refers to the ability for client connections to migrate from one server to another in the event of server failure. The client applications can continue to operate with minimal downtime.
+
+Chef Automate Cluster allows you to minimize downtime by utilizing redundant systems in case of a failure or maintenance. It includes HA capabilities for Automate web services and the associated database services, which are:
+
+- Automate Frontend
+
+Both Chef Automate and Chef Server have a load balancer with a UI. For example, let's say we have three Chef Automate and Chef Server instances. If any of the Chef Automate or Chef-Server instances fails to operate, then the traffic is distributed between the rest of the two servers. Thus, you will never experience any downtime, or the amount of downtime would be minimal.
+
+- Automate Backend
+
+The **Elasticsearch** and **Postgresql** database instances act as an automated backend component. Chef habitat's hab supervisor concept is used to make a cluster for the database instance. Automate backend cluster rests in the habitat ring. For **Postgresql**, **pgleaderchk** service runs in all the **Postgresql** instances and ensures to choose a leader in case the leader database fails. For **Elasticsearch**, there is a **msae** concept of leader-follower, and for any database failure, a leader election occurs, and a new leader is chosen.
+
+### Disaster Recovery (DR)
+
+Disaster Recovery (DR) is a comprehensive plan for recovering critical operations and systems after catastrophic events. Chef Automate HA focuses on serious but more typical failures, such as a failing component or server. A DR plan may cope with the loss of an entire region.
+
+Chef Automate HA promotes the DR cluster through regular backup and restores measures syncing the data from the production cluster to the DR cluster. Typically, these two clusters are located in different data centers or cloud provider regions enabling a production environment in a short period with minimal data loss.
+
+# Topology (Details on components- introduction and details around that) - pending - needed inputs from Eng. team
```

Comment:
Component details can be found in this document:
https://progresssoftware.sharepoint.com/:w:/s/ChefCoreC/EcQuI431G2dJpACerj638IAB9hR-JfMsu7l2uy9d9I6DGA?e=B7p8bS